mods/ra/maps/soviet-soldier-volkov-n-chitzkoi/rules.yaml

```diff
@@ -0,0 +1,204 @@
+World:
+	LuaScript:
+		Scripts: scu35ea.lua, scu35ea-AI.lua
+	MissionData:
+		Briefing: The Allies have stolen a new armor plating from us. With it, they can make structures nearly invulnerable. Their processing plant must be destroyed.\n\nFor this critical mission, we are allowing you access to Soldier Volkov. Use him and his cybernetic dog to sabotage the Allied control center, allowing our units in the southeast access to the alloy facility.\n\nThe mission is a success when Volkov destroys the control center, and our units destroy the alloy facility.\n
+		WinVideo: grvestne.vqa
+		LossVideo: sovcemet.vqa
+	ScriptLobbyDropdown@difficulty:
+		ID: difficulty
+		Label: Difficulty
+		Values:
+			easy: Easy
+			normal: Normal
+			hard: Hard
+		Default: easy
+
+Player:
+	LobbyPrerequisiteCheckbox@GLOBALBOUNTY:
+		Enabled: False
+		Locked: True
+
+MSLO:
+	Buildable:
+		Prerequisites: ~disabled
+        
+E7:
+	Buildable:
+		Prerequisites: ~disabled
+	Health:
+		HP: 25000
+	Armor:
+		Type: Heavy
+        
+2TNK:
+	Buildable:
+		Prerequisites: ~vehicles.allies, ~techlevel.medium
+        
+CA:
+	Buildable:
+		Prerequisites: ~syrd, ~techlevel.high
+        
+C2:
+	Inherits@2: ^ArmedCivilian
+        
+C3:
+	Inherits@2: ^ArmedCivilian
+        
+C4:
+	Inherits@2: ^ArmedCivilian
+        
+C5:
+	Inherits@2: ^ArmedCivilian
+        
+C6:
+	Inherits@2: ^ArmedCivilian
+        
+C8:
+	Inherits@2: ^ArmedCivilian
+        
+C9:
+	Inherits@2: ^ArmedCivilian
+        
+C10:
+	Inherits@2: ^ArmedCivilian
+        
+V01:
+	SpawnActorOnDeath:
+		Actor: healcrate
+        
+HOSP:
+	Tooltip:
+		GenericName: Hospital
+		GenericVisibility: Enemy, Ally, Neutral
+		GenericStancePrefix: false
+		TooltipDescription: Destroy this hospital to get healing equipment.
```

Comment:
This is indented wrong. `TooltipDescription` is it's own trait:

```
	TooltipDescription:
		Description: Destroy this hospital to get healing equipment.
```

should work.

Reply:
![notworking](https://user-images.githubusercontent.com/42563347/45836068-d0b13680-bcd9-11e8-8051-8a684ba735c8.PNG)
Not sure if by its own trait you mean outside of tooltip but if I'm doing it like that the game wont even load the map.

![working](https://user-images.githubusercontent.com/42563347/45836197-24238480-bcda-11e8-943e-07387bf275e6.PNG)
On the other hand if I indent it like you told me but inside tooltip the map load. Can you confirm this will work before I update the commit? Thanks you.

Reply:
[TooltipDescription](https://github.com/OpenRA/OpenRA/blob/d33ac0c8384ae360b725b5886f4c28ade95af315/OpenRA.Mods.Common/Traits/TooltipDescription.cs) is a different trait to [Tooltip](https://github.com/OpenRA/OpenRA/blob/d33ac0c8384ae360b725b5886f4c28ade95af315/OpenRA.Mods.Common/Traits/Tooltip.cs).  When you indent the definition as in your second example the game tries to load a `TooltipDescription` *field* on the `Tooltip` trait, which does not exist and is therefore ignored.

The lint check (which you can view from the "Details" link in the "Some checks were not successful" panel below reports this more explicitly:
```
OpenRA.Utility(1,1): Error: rules.yaml:75 refers to a trait field `TooltipDescription` that does not exist on `Tooltip`.
OpenRA.Utility(1,1): Error: rules.yaml:101 refers to a trait field `TooltipDescription` that does not exist on `Tooltip`.
```

Reply:
> Not sure if by its own trait you mean outside of tooltip but if I'm doing it like that the game wont even load the map.

How do you test this map? Your local OpenRA installation or this branch compiled from source?

Reply:
My local OpenRA installation.